src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala

```diff
@@ -211,7 +211,7 @@ object SchemaTools extends SparkSessionWrapper {
    * @return
    */
   private def sanitizeFieldName(s: String): String = {
-    s.replaceAll("[^a-zA-Z0-9_]", "")
+    s.replaceAll("[^a-zA-Z0-9_]", "_")
```

Comment:
my main concern here is that we may have two names of the same length that will be converted into `___` strings of the same length.

Reply:
that is possible but this is always wrapped in scrubSchema which, in the case of a duplicate there will be a unique suffix pasted. If you have ideas on how to improve this, I'm all ears. :) 

see [generateUniques](https://github.com/databrickslabs/overwatch/blob/3fb6414a8a813a5652dce9fcd256162f78202086/src/main/scala/com/databricks/labs/overwatch/utils/SchemaTools.scala#L236-L266) which is part of scrubSchema pipeline along with private function `sanitizeFields`